recipe/meta.yaml

```diff
@@ -15,12 +15,12 @@ build:
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python >=3.9
```

Comment:
Why? We need CFEP-25.

Reply:
Just because I was too lazy to figure out this build failure :) https://github.com/conda-forge/hvplot-feedstock/runs/41488465467